lib/time.js

```diff
@@ -8,12 +8,17 @@ function epochTime () {
   return ',"time":' + Date.now()
 }
 
+function unixTime () {
+  return ',"time":' + Math.round(Date.now() / 1000.0)
```

Comment:
is there a specific reason to divide by `1000.0` and not `1000`?

Reply:
Is this a blocker for you @davidmarkclements ?

Reply:
habit from strictly typed languages, don't have a strong feeling about this one. your guys call.

Reply:
I think it's a good practice to divide float with another float I get that it's not an issue in JS land.

Reply:
Unless @davidmarkclements chimes back in before I get to a computer, with time to do it, I'll merge.

Reply:
Thanks, @jsumners again your call don't have a strong feeling about this one.